incubator/hnc/Makefile

```diff
@@ -180,5 +179,36 @@ kind-deploy:
 
 ###################### RELEASE ACTIONS #########################
 # Build the container image by Cloud Build and build YAMLs locally
-release: manifests
-	gcloud builds submit --config cloudbuild.yaml .
+#
+# ALL COMMANDS THAT INCLUDE THE PERSONAL ACCESS TOKEN ARE HIDDEN
+release: check-release-env test
```

Comment:
We currently aren’t doing any logic to prevent re-pushing tags (so changing the underlying image from users). We should probably have a TODO for that as we start to move into more stable releases.

Reply:
Filed #333